docs/dev/code_format.md

```diff
@@ -1,27 +0,0 @@
-## IDE Code Format Setup
-
-### Eclipse
-
-Project preferences > Java > java format, import and select the eclipse format xml file.
-
-- [https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_format_style.xml](https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_format_style.xml)
-- [https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_cleanup.xml](https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_cleanup.xml)
-- [https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea.importorder](https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea.importorder)
-
```

Comment:
I don't see information related to where one can find the code formatter configuration files (under _eclipse/format/_) in the README.  There is a short blurb in _code_standards.md_, but it points to a broken link.  Somehow, we should note where these files reside.

Reply:
Fair enough, added a link to the section in: 188dec6a8164a4b254e9cf8d384c5535bebd58af

I reduced the verbosity, it's pretty sufficient to give a link to the folder. I hope/expect for us to clean up the IDE configuration files in the future, while here it's good to reduce the coupling between documentation and the folder structure. (The current top level has been messy for a very long time)